src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -322,6 +351,21 @@ private static List<String> parseArrayQueryParam(@Nullable String string, Charac
         : Arrays.asList(string.split(delimiter + "\\s*"));
   }
 
+  /**
+   * Return a {@link JobCreationMode} or raise an exception if the string does not match a variant.
+   */
+  private static JobCreationMode parseJobCreationMode(@Nullable String string)
```

Comment:
is this  function needed?
Just calls valueOf

Reply:
1. static method on enum 'fromString' ?
2. Do we really want to raise that exception :)